tests/testthat/test-create_target_metadata.R

```diff
@@ -62,7 +62,7 @@ test_that("create_target_metadata functions error correctly", {
         time_unit = "week"
       ),
       create_target_metadata_item(
-        target_id = "inc death",
+        target_id = "inc hosp",
```

Comment:
This change is to ensure the old test was actually testing what it was originally designed to, i.e. the presence of duplicate `target_id`/`target_key` values in a `target_metadata` block. The added check meant it was now detecting the target_id/target_key` value mismatch instead of the duplicate target_keys. I changed it so the target_id now matches the target_keys and it is now successfully picking up the duplicate `target_id`s first.